.github/workflows/assets.yml

```diff
@@ -56,6 +56,7 @@ jobs:
         with:
           token: ${{ steps.generate-token.outputs.token }}
           committer: Ghostery Adblocker Bot <ghostery-adblocker-bot@users.noreply.github.com>
+          author: Ghostery Adblocker Bot <ghostery-adblocker-bot@users.noreply.github.com>
```

Comment:
Looks like this could be the reason indeed, according to how the commit looks like.

![image](https://user-images.githubusercontent.com/1299873/149759833-34e27892-601a-4f75-86c8-49ee72fc6c17.png)